### PR TITLE
Sample apps update

### PR DESCRIPTION
**Pull request by ghost**
Added new sample apps content:
- Sole Factory
- Search Comparison
- GraphQl Caching

### CODE REVIEW

File: docs/apps/address-book.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+sidebar_position: 6
```
Comment (diana-macrometa) [inline docs/apps/address-book.md:2]: Not something to necessarily change right now, but I strongly recommend at least two-digit sidebar position numbers (10, 20, 30) to start. That way, you can insert items later (i.e. 25) without renumbering.

File: docs/apps/graph-ql-caching.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 
+
+## Solution
+Immerse yourself in a comprehensive demonstration that illuminates the tremendous strides made in optimizing GraphQL query response times while effectively alleviating the burden on the GraphQL origin server. Our demo showcases a significant architectural enhancement achieved by customizing the Quell Library (https://github.com/open-source-labs/Quell) to leverage GDN KV collection as a superior caching mechanism, surpassing traditional Redis server implementations. By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region, we unlock unparalleled performance gains. Prepare to delve deep into the intricacies of this cutting-edge setup, unraveling the inherent power of GDN in revolutionizing GraphQL workflows.
```
Comment (tushar-macrometa) [inline docs/apps/graph-ql-caching.md]: @dlozina-macrometa can we please remove setup details from solution text?
```
Remove:
By meticulously configuring a robust 4-region US federation with GDN and deploying a single Elasticsearch instance strategically located in the US West region,
```

Comment (ghost) [reply docs/apps/graph-ql-caching.md]: @tushar-macrometa sure!

File: docs/apps/graph-ql-caching.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 3
+title: GraphQl Caching
+---
+
+## Macrometa GDN
+
+| **Platform**                       | **Tenat**                      |
```
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]:
```suggestion
| **Platform**                       | **Tenant**                      |
```

File: docs/apps/graph-ql-caching.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 3
+title: GraphQl Caching
+---
+
+## Macrometa GDN
```
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md:10]: I really hate having the tenant information in our docs. What value does it provide to our end user?

File: docs/apps/graph-ql-caching.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 3
+title: GraphQl Caching
```
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]:
```suggestion
title: GraphQL Caching
```

File: docs/apps/graph-ql-caching.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+| **Platform**                       | **Tenat**                      |
+| ---------------------------------- | ------------------------------ |
+| [URL](https://cache.eng.macrometa.io/) | `demo@macrometa.com` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) |
```
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]: This GitHub repo is private. This is not useful for our users.

Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]:
```suggestion
| [GitHub](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 
```

File: docs/apps/graph-ql-caching.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+| ---------------------------------- | ------------------------------ |
+| [URL](https://cache.eng.macrometa.io/) | `demo@macrometa.com` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/parsec-graphql-cache) | [Deployment](https://macrometacorp.github.io/parsec-graphql-cache/) | 
+
+## Solution
```
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]:
```suggestion
## Solution

```

File: docs/apps/graph-ql-caching.md (hunk 5 above)
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]: Will our users know what GraphQL is? I don't think it's an industry term.

File: docs/apps/graph-ql-caching.md (hunk 2 above)
Comment (diana-macrometa) [inline docs/apps/graph-ql-caching.md]:
```suggestion
This sample application demonstrates how to optimize GraphQL query response times while reducing the burden on the GraphQL origin server. This demo includes a customized Quell Library (https://github.com/open-source-labs/Quell) to leverage a Macrometa key-value collection as a caching mechanism, surpassing traditional Redis server implementations. We achieve performance gains by configuring a four-region US federation with GDN and deploying a single Elasticsearch instance located in the US West region. 
```

Comment (diana-macrometa) [reply docs/apps/graph-ql-caching.md]: Wrong tone for technical documentation. It sounds like you swallowed a salesman.

File: docs/apps/search-comparison.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+| **Platform**                       | **Tenat**                      | **Geo Fabric** |
+| ---------------------------------- | ------------------------------ | -------------- |
+| [Play](https://play.macrometa.io/) | `demo@macrometa.com` | `search_comparison` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/demo-search-comparison) | [Deployment](https://macrometacorp.github.io/demo-search-comparison/) |
```
Comment (diana-macrometa) [inline docs/apps/search-comparison.md]: Again, this is a private repo, and I still hate the tenant information in our docs.

File: docs/apps/search-comparison.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 2
+title: Search Comparison
+---
+
+## Macrometa GDN
+
+| **Platform**                       | **Tenat**                      | **Geo Fabric** |
```
Comment (diana-macrometa) [inline docs/apps/search-comparison.md]:
```suggestion
| **Platform**                       | **Tenant**                      | **GeoFabric** |
```

File: docs/apps/search-comparison.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/demo-search-comparison) | [Deployment](https://macrometacorp.github.io/demo-search-comparison/) | 
+
+## Solution
+
+Our objective in this demo is to provide a comprehensive analysis of latency variations between GDN search and ElasticSearch, shedding light on their contrasting performance. Leveraging a meticulously curated sample dataset featuring detailed information on Samsung products, we present an in-depth exploration of search technology capabilities. Prepare to delve into the intricacies of latency as we dissect and compare the responsiveness of these two powerful search solutions. Gain valuable insights into the inner workings of each system, unlocking a deeper understanding of their engineering prowess and potential applications.
```
Comment (diana-macrometa) [inline docs/apps/search-comparison.md]:
```suggestion
This demo provides a comprehensive analysis of latency variations between Macrometa search views and ElasticSearch, contrasting their performance. Leveraging a curated sample dataset featuring detailed information on Samsung products, we present an in-depth exploration of search technology capabilities.
```

File: docs/apps/sole-factory.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 1
+title: Macrometa Sole Factory
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]: I'm confused by the naming. The first two seemed to be named based on features you are demonstrating, this one is based on what it is. Be consistent.

File: docs/apps/search-comparison.md (hunk 8 above)
Comment (diana-macrometa) [inline docs/apps/search-comparison.md]:
```suggestion
| [GitHub](https://github.com/Macrometacorp/demo-search-comparison) | [Deployment](https://macrometacorp.github.io/demo-search-comparison/) | 
```

File: docs/apps/sole-factory.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+| **Platform**                       | **Tenat**                      | **Geo Fabric** |
+| ---------------------------------- | ------------------------------ | -------------- |
+| [Play](https://play.macrometa.io/) | `demo@macrometa.com` | `e-commerce-search-flow` |
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) |
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]:
```suggestion
| [GitHub](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) | 
```

Comment (diana-macrometa) [reply docs/apps/sole-factory.md]: Private repo.

File: docs/apps/sole-factory.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+## Deployment
+
+| **Source Code**                       | **Deployment**                      | 
+| ---------------------------------- | ------------------------------ | 
+| [Github](https://github.com/Macrometacorp/e-commerce) | [Deployment](https://macrometacorp.github.io/e-commerce/) | 
+
+## Solution
+
+Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. Discover a fully functional e-commerce website that offers personalized experiences to every visitor. Our cutting-edge technology tracks and analyzes visitor actions, enabling us to provide tailored recommendations based on their preferences. Furthermore, our e-commerce store boasts real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]:
```suggestion
Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. This sample application is a fully functional e-commerce website that offers personalized experiences to every visitor. Using the Macrometa platform, it tracks and analyzes visitor actions to provide tailored recommendations based on their preferences. The e-commerce store has real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.
```

File: docs/apps/sole-factory.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+## Components
+
+This demo includes the following components:
+
+- Personalized recommendations - Macrometa Stream Workers read visitor actions from a Collection Stream, creating a list of recommended products for each visitor.
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]: Nothing in this list should be capitalized except Macrometa and the first word in the sentence or definition term. Remove GDN.

File: docs/apps/sole-factory.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+- Real-time stock updates - Macrometa Stream Workers read product updates from a Collection Stream, updating the product stock in real-time.
+- Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand and category.
+- Serverless backend – Macrometa Query Workers provides an interface between the web app and the backend, providing ultra-low latency data and compute.
+
+## Visitor Personalization
+
+Macrometa GDN Streams and Query Workers enable real-time visitor personalization, including recommendations and real-time stock updates.
+
+- **Tailored Product Recommendations**
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]: Remove the bold.

File: docs/apps/sole-factory.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+This demo includes the following components:
+
+- Personalized recommendations - Macrometa Stream Workers read visitor actions from a Collection Stream, creating a list of recommended products for each visitor.
+- Real-time stock updates - Macrometa Stream Workers read product updates from a Collection Stream, updating the product stock in real-time.
+- Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand and category.
+- Serverless backend – Macrometa Query Workers provides an interface between the web app and the backend, providing ultra-low latency data and compute.
+
+## Visitor Personalization
+
+Macrometa GDN Streams and Query Workers enable real-time visitor personalization, including recommendations and real-time stock updates.
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]:
```suggestion
Macrometa streams and query workers enable real-time visitor personalization, including recommendations and real-time stock updates.
```

File: docs/apps/sole-factory.md [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+
+Experience the full potential of Macrometa's advanced full-stack capabilities with our innovative e-commerce demo. Discover a fully functional e-commerce website that offers personalized experiences to every visitor. Our cutting-edge technology tracks and analyzes visitor actions, enabling us to provide tailored recommendations based on their preferences. Furthermore, our e-commerce store boasts real-time stock updates and features a sophisticated search engine, ensuring a seamless and efficient shopping experience.
+
+## Components
+
+This demo includes the following components:
+
+- Personalized recommendations - Macrometa Stream Workers read visitor actions from a Collection Stream, creating a list of recommended products for each visitor.
+- Real-time stock updates - Macrometa Stream Workers read product updates from a Collection Stream, updating the product stock in real-time.
+- Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand and category.
```
Comment (diana-macrometa) [inline docs/apps/sole-factory.md]:
```suggestion
- Search - Macrometa GDN Search service enables full-text search for our storefront, enabling users to find products based on a variety of terms including product name, brand, and category.
```

**Review by diana-macrometa — DISMISSED** [review]

**Comment by diana-macrometa** [comment]
@dlozina-macrometa, are we planning to make the demo repositories public? If not, then they do not belong in our user documentation.

**Comment by elof** [comment]
@diana-macrometa following up on this kinda late, but I don't think we should put sample apps that have private code should be listed in Sample Apps. Maybe we could have a section that lists public demos if we want to have something people can interact with but can't access the code? cc @dlozina-macrometa 

Another discussion point is are the demos/sample apps that don't have their code public that can/should?

**Comment by diana-macrometa** [comment]
> @diana-macrometa following up on this kinda late, but I don't think we should put sample apps that have private code should be listed in Sample Apps. Maybe we could have a section that lists public demos if we want to have something people can interact with but can't access the code? cc @dlozina-macrometa
> 
> Another discussion point is are the demos/sample apps that don't have their code public that can/should?

@elof, agreed. Private apps should not be in the docs, and neither should the info for our internal demo login. I wasn't sure if the repos were private termporarily while they were being developed or not.

**Comment by ghost** [comment]
@diana-macrometa, @elof this was Durga's request. He wanted us to add demos to sample apps. I agree with Justin; for private repos, we should have a different section containing only links to GitHub pages so people can interact with the demos and use the features.
We can remove credentials and links to the federation and move just explanation and Github pages links to a new section. 
cc @diana-macrometa

**Comment by diana-macrometa** [comment]
Love it, @dlozina-macrometa. If we can hide the private demos from casual viewers, that would be even better.

**Comment by diana-macrometa** [comment]
I am taking over this PR. Reviewers, please review.

**Review by elof — APPROVED** [review]
:shipit:

**Pull request merged**